frontend/src/static/js/v_zoom.js

```diff
@@ -1,3 +1,8 @@
+const commitSortDict = {
+  lineOfCode: (commit) => commit.insertions,
```

Comment:
why this dict can not be a data of `vZoom`?

Reply:
Vue sets all the properties in the data to setters/getters to make them reactive and I feel this dict should be static.

Reply:
OK, got what you mean.